### PR TITLE
3x tweaks

### PR DESCRIPTION
**Pull request by manolama**

### CODE REVIEW

File: storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java [hunk 1]
```diff
@@ -87,56 +105,54 @@
...
+  protected List<byte[]> tsuids;
   
   /** The number of get requests batches allowed to be outstanding at
    * any given time. */
-  protected final int concurrency_multi_get;
+  protected int concurrency_multi_get;
   
   /** Whether or not we're fetching rows in reverse. */
-  protected final boolean reversed;
+  protected boolean reversed;
```
Comment (sidhhu) [inline storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java]: why?

Comment (manolama) [reply storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java]: Cause it's a pooled object now so this has to be reset on each query.

Comment (sidhhu) [reply storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java]: i mean why use reversed at all.

Comment (manolama) [reply storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java]: oh cause we will eventually stream data from most recent to oldest so we can load like Splunk does.

**Pull request merged**